ligo/main.mligo

```diff
@@ -365,6 +365,19 @@ let update_timed_cumulatives (s : storage) : storage =
         }
         in {s with cumulatives_buffer = new_buffer}
 
+let get_position_info (s : storage) (p : get_position_info_param) : result =
+    let position_index = match Big_map.find_opt p.position_id s.position_indexes with
+        | None -> (failwith position_not_exist_err : position_index)
+        | Some v -> v in
```

Comment:
It looks like we have `get_position_index` for this, it returns an FA2-specific error, but IMO that's still valid here.

Reply:
Oh nice observation, I missed the existence of `get_position_index`.

In the solidity implementation of liquidity mining contract they treat positions very much like tokens, even call the respective variable `token_id`, not `position_id`, so I agree that having FA2-specific error can be quite justified here.